src/MongoDB.Driver.Core/TargetFramework.cs

```diff
@@ -0,0 +1,29 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+namespace MongoDB.Driver.Core
```

Comment:
It's unfortunate that most of our projects (Driver, Driver.Core, Driver.Legacy) have the same namespace. I've added a postfix (`.Core`) to be able to use in tests projects which have links on all of the solution projects.

Reply:
I would consider changes in namespaces as part of work for `3.0` (if we will still have these 3 projects in the driver in 3.0)

Reply:
Feel free to add it to the planning doc if it's not already there!
https://docs.google.com/document/d/1NMNLpmlQa0rOHDHX4sZ1cjShPvqvOjwBAlXNHzKfaNI/edit :)

Reply:
The different projects use the same namespace for many (not all) classes for backward compatibility with 1.x and to facilitate sharing classes between High and Core without too many `using` directives.

Depending on the scope of 3.0 we may or may not want new namespaces. It would certainly be a breaking change.

Reply:
The namespace of this particular class should mirror the name of the DLL, so this is correct.

We want `MongoDB.Driver.Core.TargetFramework.Moniker` to be the target framework of the `MongoDB.Driver.Core.dll` file.